src/main/java/com/spring/user/api/dto/UserDto.java

```diff
@@ -0,0 +1,22 @@
+package com.spring.user.api.dto;
+
+import javax.validation.constraints.Email;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class UserDto {
+	
+	private Long id;
+	private String firstname;
+	private String middlename;
+	private String lastname;
```

Comment:
The other fields should have validation annotations as well. For example, `firstname` can not be empty.